timer/timer.c

```diff
@@ -11,6 +11,7 @@
 #define DOT_POS 9
```

Comment:
Is this define used after code change?

Reply:
No, it's useless. I forgot to remove this define making changes